### PR TITLE
Add multi-tenancy whitelist info.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## PR description
Add information about how to limit access using the `whitelist` trust mode.

### CODE REVIEW

**Review by MadelineMurray — APPROVED** [review]
LGTM but will need a tech review by someone before merging.

**Review by jframe — APPROVED** [review]
LGTM

**Pull request merged**
